spack/packages/dla-future/package.py

```diff
@@ -33,18 +33,26 @@ class DlaFuture(CMakePackage, CudaPackage):
 
     depends_on("umpire~examples")
     depends_on("umpire+cuda~shared", when="+cuda")
+    depends_on("umpire+rocm~shared", when="+rocm")
 
     # https://github.com/eth-cscs/DLA-Future/issues/420
     conflicts("umpire@6:")
 
     depends_on("pika cxxstd=17 +mpi")
-    depends_on("pika@0.4:")
+    depends_on("pika@0.5:")
```

Comment:
To do before merging:
```suggestion
    depends_on("pika@0.6:")
```